### ISSUE TITLE
Reconsider the use of ORCID: in LinkML Tutorial

### ISSUE THREAD

**Issue by hsolbrig** [linkml/linkml#1042]
tldr;
1) We shouldn't use ORCID in the id field in the examples until we are equipped to deal with its nuances and
2) We should consider fixing multiple issues described below.

The [LinkML tutorial](https://linkml.io/linkml/intro/tutorial01.html) uses a curie-like identifier in the data:
`id: ORCID:1234`

This raises a couple of issues:
1) (The easy one) "1234" is not a valid ORCID.
2) There is a strong implication that we should be treating this is a curie.  

When I _try_ to define `id` as a `uriorcurie`, I encounter a whole pile of issues, all of which definitely need addressing, including:
1) I tried to do what seemed obvious, which was:
```
id: https://w3id.org/linkml/examples/personinfo
name: personinfo
prefixes:
    linkml: https://w3id.org/linkml/
    ORCID: https://orcid.org/
imports:
  - linkml:types
default_range: string

classes:
  Person:
    attributes:
      id:
        type: uriorcurie
      full_name:
      aliases:
      phone:
      age:
```
This didn't work because `type`, while seemingly intuitive, isn't how one defines the "type" of an attribute.

2)
```
id: https://w3id.org/linkml/examples/personinfo
name: personinfo
prefixes:
  linkml: https://w3id.org/linkml/
  ORCID: https://orcid.org/
imports:
  - linkml:types
default_range: string

classes:
  Person:
    attributes:
      id:
        range: uriorcurie
      full_name:
      aliases:
      phone:
      age:
```
This fails with a horrible error that only a LinkML insider could know and love.  The output requires the `xsd:` namespace and, despite the fact that it is OUR CODE that inserted it, we still require the user to add it to the model.  Ick!
3) 
```id: https://w3id.org/linkml/examples/personinfo
name: personinfo
prefixes:
  linkml: https://w3id.org/linkml/
  ORCID: https://orcid.org/
  xsd: http://www.w3.org/2001/XMLSchema#
imports:
  - linkml:types
default_range: string

classes:
  Person:
    attributes:
      id:
        range: uriorcurie
      full_name:
      aliases:
      phone:
      age:
```
Finally, we have a schema that works but yields
```
> linkml-convert -s personinfo.yaml data.yaml -t rdf
@prefix ns1: <https://w3id.org/linkml/examples/personinfo/:> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .

[] a ns1:Person ;
    ns1:age "32" ;
    ns1:full_name "Clark Kent" ;
    ns1:id "ORCID:0000-0002-0860-8606"^^xsd:anyURI ;
    ns1:phone "555-555-5555" .
```
Where the ORCID curie is neither recognized nor processed.

**Comment by cmungall** [comment]
We can definitely use ORCIDs that are conformant with ORCID syntax - or just use a fake pseudo-orcid system, no string opinions.

Regarding

```yaml
id:
        type: uriorcurie
```

I'm not sure what we can do about this, this isn't sanctioned in the metamodel. We should definitely have clear reporting of errors. We could even have some kind of error system that suggests repairs... but this seems like a new nice to have feature, and less necessary as we move towards workflows with either schema-enabled IDEs or web based editing..

The xsd declaration issue seems like a code thing rather than a documentation thing, we should fix that

Note to get your desired behavior you need to declare id as an identifier. 

The intended behavior of range: uriorcurie for non-identifiers is an interesting one. There are valid arguments to be made for defaulting to anyURI (if you want a node, make the range a possibly non-inlined reference). Maybe we also want to introduce a new type, note that uri is already mapped to xsd:anyURI 

```yaml
  uri:
    uri: xsd:anyURI
    base: URI
    repr: str
    description: a complete URI
```

lots to discuss here! This is super-useful, thanks!

**Comment by sierra-moxon** [comment]
Goal: have the documentation use the ORCID and have it work correctly.  Someone could just start editing the tutorial, fixing up the uriorcurie issue if we could.

**Comment by cmungall** [comment]
OK, a lot going on here.

I think the first mistake is probably introducing RDF conversion too early in the tutorial. To get useful RDF output we first need to introduce multiple different concepts:

- prefixes
- ranges
- types, and the uri or uriorcurie type
- identifier

The original idea was to start with the simplest example possible, and just to show that RDF generation was possible. But this is introducing too much confusion. Let's just take the RDF example out of part 1, and do it justice in [part 4](https://linkml.io/linkml/intro/tutorial04.html)

Going back to Harold's other points:

1. `type` is not a valid metamodel slot. We could make the reporting here clearer. We could even have it say something like 'no such slot as "type", did you mean "range"', as this is likely to be a common mistake
2. some more intuitive reporting would definitely be better here

Regarding

> Where the ORCID curie is neither recognized nor processed.

I think your intention here is that the ORCID becomes the IRI of the person. This is reasonable. However, linkml doesn't know that you intend `id` to be the identifier field any more than you intend `phone` to be the identifier field. We could potentially linter warn if you are using a slot name like `id` in a non-idiomatic way but this shouldn't be enforced.

But I think most of this is clarified by moving the RDF section later
